### ISSUE TITLE
[Feature proposal] Noew commands \coloniff etc.

### ISSUE THREAD

**Issue by siebenstreich** [latex3/mathtools#59]
The `mathtools` package gives us commands like `\coloneqq` and `\eqqcolon` for typographically sound symbols for definitions of expressions via equality sign. However, the natural counterpart for definitions of statements via the equivalence sign is not there. I think it would only be natural to add analogous commands like `\coloniff`, `\iffcolon` and maybe also `\colonLeftrightarrow`, `\Leftrightarrowcolon`, `\colonLongleftrightarrow`, `'Longleftrightarrowcolon`.

**Comment by siebenstreich** [comment]
Example: On the set of positive integers, we define a partial order by
x ≤ y :⇔ x divides y

On both sides of the ":⇔"-sign, there are logical statements, which are commonly connected with symbols like  ⇔ and ⇒. So the usage of ":=" would not be correct here. Quite a few authors are using the ":⇔"-symbol in this situation.

I don't know if this symbol is in unicode. Probably it is not, unfortunately.

**Comment by davidcarlisle** [comment]
It's not a notation I've seen, of course there is no reason authors can't define it if they need it but I don't see that it's needed in a core package like mathtools. Unicode has := and =: and not any of the other suggested symbols.

**Comment by car222222** [comment]
But my argument above is no reason for us not to provide (maybe via mathtools) a method for naming (TeX names) and easily defining new "compound math symbol commands" for such notations.

But possibly we already do have such helper functions: maybe check out all the math font stuff?

**Comment by FrankMittelbach** [comment]
@car222222 yes there could be such a declaration helper and probably there should be one. If we think about accessibility of math such a declaration would also have to cover specifying what that command should do with respect to accessibility, given that the notation is user-edefined.  

However, for precisely this reason I don't think any such declaration helper should be defined right now but only after we have gained experience with providing accessible math and understand how a syntax for specifying accessibility of notation could or should look like. Doing it now, would mean the syntax would change in the near future. I would therefore think it is best to park this for now.

**Comment by car222222** [comment]
On the whole I agree re the timing and the relevance of accessibility, especially the navigation of math formulas as this is very useful for all of us!

But we, in some sense, could start thinking about this now, together the very many other aspects of providing really useful, in very many ways, math output.
